examples/karpenter/main.tf

```diff
@@ -141,7 +141,7 @@ module "karpenter_disabled" {
 
 resource "helm_release" "karpenter" {
   namespace           = "kube-system"
-  name                = "karpenter"
+  name                = module.karpenter.service_account
```

Comment:
this is not the correct place for this. if you wanted to achieve the same outcome, you would need to use this value down in the `values` and set it via https://github.com/aws/karpenter-provider-aws/blob/9b0bb2b6b3daf6b80d39fcc8c244ece76e49599d/charts/karpenter/values.yaml#L20

Reply:
@bryantbiggs oh that's right, I pushed a fix for this, could you take a look?